src/views/OrderDetailUpdated.vue

```diff
@@ -475,15 +481,34 @@ export default defineComponent({
 
       assignPickerModal.onDidDismiss().then(async(result: any) => {
         if(result.data.selectedPicker) {
-          this.pickers = result.data.picker
+          const selectedPicker = result.data.picker
+          this.pickers = selectedPicker
           this.picklistDate = DateTime.now().toMillis()
+          this.order.pickers = selectedPicker.name
+          this.order.pickerIds = [selectedPicker.id]
           await this.store.dispatch('order/packShipGroupItems', { order, part, facilityId, selectedPicker: result.data.selectedPicker })
           this.prepareOrderTimeline();
         }
       })
 
       return assignPickerModal.present();
     },
+    async editPicker(order: any) {
+      const editPickerModal = await modalController.create({
+        component: EditPickerModal,
+        componentProps: { order }
+      });
+
+      editPickerModal.onDidDismiss().then((result) => {
+        if(result.data?.selectedPicker){
+          const selectedPicker = result.data.selectedPicker
+          this.order.pickers = selectedPicker.name
+          this.order.pickerIds = [selectedPicker.id]
+          this.store.dispatch('order/updateCurrent', { order: this.order })
```

Comment:
Once the picker information is updated, prepare the order timeline again, so that it displays the updated picker information.